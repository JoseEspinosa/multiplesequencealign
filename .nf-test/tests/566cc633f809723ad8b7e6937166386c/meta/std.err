The following invalid input values have been detected:

* --outdir: expected type: String, found: Integer (1)

 -- Check script '/home/luisasantus/Desktop/multiplesequencealign/./subworkflows/nf-core/utils_nfvalidation_plugin/tests/../main.nf' at line: 57 or see '/home/luisasantus/Desktop/multiplesequencealign/.nf-test/tests/566cc633f809723ad8b7e6937166386c/meta/nextflow.log' file for more details
