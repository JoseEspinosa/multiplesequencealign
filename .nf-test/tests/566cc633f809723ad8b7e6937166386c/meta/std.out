ERROR ~ ERROR: Validation of pipeline parameters failed!

 -- Check '/home/luisasantus/Desktop/multiplesequencealign/.nf-test/tests/566cc633f809723ad8b7e6937166386c/meta/nextflow.log' file for details
